src/main/java/com/sleekbyte/tailor/output/ViolationMessage.java

```diff
@@ -38,6 +39,39 @@ public ViolationMessage(Rules rule, int lineNumber, int columnNumber, Severity s
     }
 
     /**
+     * Constructs a ViolationMessage where the rule name is replaced by tool name (tailor).
+     *
+     * @param lineNumber       the logical line number in the source file
+     * @param columnNumber     the logical column number in the source file
+     * @param severity         the severity of the violation message
+     * @param violationMessage the description of the violation message
+     */
+    public ViolationMessage(int lineNumber, int columnNumber, Severity severity, String violationMessage) {
```

Comment:
Can we move these 2 constructors either before or after the existing ones? It would be better to group the constructors so that the ones accepting `Rules` objects are side-by-side.